src/core/dbus-execute.c

```diff
@@ -1244,6 +1263,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("LogRateLimitBurst", "u", bus_property_get_unsigned, offsetof(ExecContext, log_ratelimit_burst), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogExtraFields", "aay", property_get_log_extra_fields, 0, SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogNamespace", "s", NULL, offsetof(ExecContext, log_namespace), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("CgroupNFTSet", "(iss)", property_get_cgroup_nft_set, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
in the properties we so fare spelt cgroup as "ControlGroup", please do so here too

Also, given this is a cgroup property this should really be part of CGroupContext, not ExecContext. i.e scope units should have that (as they have CGroupContext, but no ExecContext.